docker/elasticsearch/README.md

```diff
@@ -0,0 +1,29 @@
+#### 2.0 Start Elasticsearch with docker
+If you familier with docker you can use the [elasticsearch docker image](https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html) with installed elasticsearch-vector-scoring plugin.
+Just go to docker/elasticsearch/ directory where you can find the ```Dockerfile``` and build the image with
+```docker build -t es-with-vector-scoring .``` command. 
+Check that the image is successfully built:
+```
+$ docker build -t es-with-vector-scoring .
+  Sending build context to Docker daemon   5.12kB
+  Step 1/3 : FROM          docker.elastic.co/elasticsearch/elasticsearch:5.3.0
+  ...
+  Successfully built 53ee57082cb8
+  Successfully tagged es-with-vector-scoring:latest
+```
+
+
+If builds ended you can start the image with ```docker run -p 9200:9200 -p 9300:9300 es-with-vector-scoring``` command. 
+Check that the `elasticsearch-vector-scoring-plugin` is successfully loaded and elasticsearch is started:
```

Comment:
Perhaps put "Elasticsearch is started" first, i.e. "Check that Elasticsearch has started and the ... "